src/main/java/com/stripe/net/ApiResourceTypeAdapterFactoryProvider.java

```diff
@@ -18,6 +18,7 @@ final class ApiResourceTypeAdapterFactoryProvider {
     factories.add(new BalanceTransactionSourceTypeAdapterFactory());
     factories.add(new ExternalAccountTypeAdapterFactory());
     factories.add(new PaymentSourceTypeAdapterFactory());
+    factories.add(new ReflectionCheckingTypeAdapterFactory());
```

Comment:
????

Reply:
ahh, this file is apparently codegenned <3